components/proxy/Dockerfile

```diff
@@ -5,9 +5,23 @@ FROM $IMAGE_NAME:$IMAGE_VERSION
 LABEL maintainer="Quality-time team <quality-time@ictu.nl>"
 LABEL description="Quality-time proxy"
 
+ENV NGINX_LISTEN 80
```

Comment:
How does this relate to the `PROXY_PORT` [env var](https://quality-time.readthedocs.io/en/latest/software.html#id10); do they both need to be set separately, or is the alternate proxy port rerouted by ingress?

Reply:
The variable NGINX_LISTEN is configuration of Nginx in the docker container, the PROXY_PORT can be used as input for for this variable.